public/src/components/student/CoursePanel.tsx

```diff
@@ -30,7 +30,7 @@ export class CoursePanel extends React.Component<IPanelProps> {
                             selector={(item: ISubmissionLink) => {
                                 let score = "N/A";
                                 if (item.submission) {
-                                    score = item.assignment.getSkiptests() ? scoreFromReviews(item.submission.reviews).toString() : item.submission.score.toString();
+                                    score = item.assignment.getReviewers() > 0 ? scoreFromReviews(item.submission.reviews).toString() : item.submission.score.toString();
```

Comment:
This code could be converted to a helper function, including the preceding `if` statement. The same code appears also in `student/SingleCourseOverview.tsx`. But note that one case uses `toString()`, while the other doesn't, so this should be made consistent.

Reply:
See also below. Seems like the helper method could be something like `item.getScore()` or `getScore(item)`, where `item` is the `ISubmissionLink` type. This could then be reused several places.

Reply:
I move all the score calculation to the server side, this way a submission score will always be updated when a review score is updated (maybe we can even drop the `score` field on the review altogether), so I just replace all these occasions with using the Submission score directly.

Reply:
This is a good choice.